spring-rabbit/src/main/resources/org/springframework/amqp/rabbit/config/spring-rabbit-1.5.xsd

```diff
@@ -994,7 +994,23 @@
 				<xsd:annotation>
 					<xsd:documentation><![CDATA[
 	A reference to a <queue/> for replies; optional; if not supplied, methods expecting replies
-	will use a temporary, exclusive, auto-delete queue.
+	will use a temporary, exclusive, auto-delete queue or, if the RabbitMQ version is 3.4 or higher,
+	using the 'direct reply-to' mechanism. See also 'reply-address'.
```

Comment:
Yes, I originally had this as deprecated but, of course, it's needed for the listener part.